network/src/protocols/test.rs

```diff
@@ -117,6 +117,7 @@ fn net_service_start(name: String) -> Node {
         bootnode_mode: true,
         max_send_buffer: None,
         sync: SyncConfig::default(),
+        reuse: true,
```

Comment:
I prefer to this style:
https://github.com/nervosnetwork/ckb/blob/fc94d8c8468b676000a33fa8dc415ac1761a86d5/db-migration/src/lib.rs#L202-L205